js_test.go

```diff
@@ -939,6 +941,21 @@ func TestJetStreamExpiredPullRequests(t *testing.T) {
 			t.Fatalf("Expected error and wait for 250ms, got err=%v and dur=%v", err, dur)
 		}
 	}
+
+	t.Run("MaxWait timeout should return nats error", func(t *testing.T) {
```

Comment:
Could you move this test block to be as part of this other test? https://github.com/nats-io/nats.go/blob/main/test/js_test.go#L6869

Reply:
Sure :D